src/features/views/components/ViewDataTable/columnTypes/index.ts

```diff
@@ -26,7 +28,7 @@ export interface IColumnType<
   getColDef(
     column: ColumnType,
     accessLevel: ZetkinObjectAccess['level'] | null,
-    state?: RootState,
+    tagListState?: RemoteList<ZetkinTag>,
```

Comment:
The problem with this change is that it makes `IColumnType.getColDef` less polymorphous. The purpose of polymorphism here is that different column types may need different parts of the state.

You are right that the only state ever used are the tags, but that's only because that's the only column that _currently_ needs state. Other columns might need more state in the future.

With that said, the pattern we have used here is the last remnant of a previous experiment with state management inspired by Flutter/Dart, that did not turn out great for React. It could do with a more substantial refactor, so I'm not sentimental at all about keeping the polymorphism for potential future columns, instead hoping that we will have refactored it away entirely by then.